bottle_app.py

```diff
@@ -60,6 +60,7 @@
 
 import db
 from auth import get_user_api_key, get_user_ipaddr, get_movie_id, API_KEY_COOKIE_NAME
+import paths
 from paths import view,STATIC_DIR,TEMPLATE_DIR,PLANTTRACER_ENDPOINT
 from lib.ctools import clogging
```

Comment:
The imports in this module are getting a little messy.

1. Does it make style sense to have both a direct import and a from import referencing the same module?
2. [PEP8 Imports](https://pep8.org/#importsl) would like us to have our imports in this order and in sections: standard packages, then 3rd party, then local, with each section separated by a blank line and each sections imports in alphabetical order.

Reply:
There is a program called autopep8 which I will run. I think that it's okay to have import for the named constants and also a generic import.